modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm

```diff
@@ -751,40 +751,40 @@
 
 // Donation reward for CandleJax
 /obj/item/clothing/head/helmet/space/plasmaman/candlejax
-	name = "emission's helmet"
-	desc = "A special containment helmet designed for heavy usage. Multiple dings and notches are on this one."
+	name = "Specialised helmet"
+	desc = "A specialised helmet designed for heavy usage. Designed for Plasmamen who fancy a new look."
 	icon = 'modular_skyrat/master_files/icons/donator/obj/clothing/hats.dmi'
 	worn_icon = 'modular_skyrat/master_files/icons/donator/mob/clothing/head.dmi'
 	icon_state = "emissionhelm"
 
 
 // Donation reward for CandleJax
 /obj/item/clothing/head/helmet/space/plasmaman/candlejax2
-	name = "azulean's environment helmet"
-	desc = "An Azulean-made Enviro-Helmet, adjusted for the unique skull shape typical of the species. Alongside the standard features, it includes an embossment of the Azulean Crest on the back of the helmet."
+	name = "Skullshape environment helmet"
+	desc = "A new Enviro-Helmet, adjusted for the unique skull shape typical of some Plasmamen."
 	icon = 'modular_skyrat/master_files/icons/donator/obj/clothing/hats.dmi'
 	worn_icon = 'modular_skyrat/master_files/icons/donator/mob/clothing/head.dmi'
 	icon_state = "anahead"
 
 // Donation reward for CandleJax
 /obj/item/clothing/under/plasmaman/candlejax
-	name = "emission's containment suit"
-	desc = "A modified envirosuit featuring a reserved color scheme."
+	name = "Specialised suit"
+	desc = "A modified envirosuit featuring a reserved color scheme for plasmamen looking to try on something new."
 	icon = 'modular_skyrat/master_files/icons/donator/obj/clothing/uniform.dmi'
 	worn_icon = 'modular_skyrat/master_files/icons/donator/mob/clothing/uniform.dmi'
 	icon_state = "emissionsuit"
 
 // Donation reward for CandleJax
 /obj/item/clothing/under/plasmaman/candlejax2
-	name = "azulean's environment suit"
-	desc = "An Azulean-made Enviro-Suit. Fitted to the Azulean form, it has surplus containment fabric designed to give the solidified mass of plasma that was once a tail some breathing room."
+	name = "skeleton shame environment suit"
+	desc = "An Enviro-Suit. Fitted to the the form of some other species who fell into plasma."
```

Comment:
1. what the fuck is the item name even supposed to mean? just call it something normal
2. "fitted to the form of other species" fits a lot better and doesnt sound rude (oh yeah just _some other_ species)

Reply:
This is a typo. The item is for a plasma man with a tail, so it was some species other than human, and yeah, plasma men are sorta uncared for so the description DOES reflect that. Still, good catch on the typo!